conda-forge.yml

```diff
@@ -1,11 +1,9 @@
 conda_forge_output_validation: true
-bot:
-  automerge: false
 github:
   branch_name: main
   tooling_branch_name: main
 conda_build:
   pkg_format: '2'
 bot:
-  automerge: true
-  inspection: update-all
\ No newline at end of file
+  automerge: false
```

Comment:
Do you all want true or false here? We had both 😬

Reply:
True :)

Reply:
Done!